frontend/src/features/account/Login.tsx

```diff
@@ -31,6 +25,8 @@ const Login = () => {
   const activated = useSelector<RootState, IGenericNetworkAction>(
     state => (state.network as any)[actionTypes.ADD_ACTIVATE_USER] as IGenericNetworkAction,
   );
+  const { login } = tenant();
```

Comment:
premature optimization, but having to re-evaluate tenant every time it is used isn't ideal given that the environment variables shouldn't change when the application is running. This could be memoized or changed to a globally executed function and only store/export the result.

Reply:
I thought of this, but wasn't sure the best approach in this case.  I see what I can do to improve this.

Reply:
Take another look at my implementation.  I think it is much better now.